packages/notifi-dapp-example/src/app/notifi/dashboard/page.tsx

```diff
@@ -1,58 +1,68 @@
 'use client';
 
-import { useChain } from '@cosmos-kit/react';
-import { useNotifiClientContext } from '@notifi-network/notifi-react-card';
-import localforage from 'localforage';
+import { AlertSubscription } from '@/components/AlertSubscription';
+import { DashboardSideBar } from '@/components/DashBoardSideBar';
+import { HistoryDetail } from '@/components/HistoryDetail';
+import { HistoryList } from '@/components/HistoryList';
+import { useGlobalStateContext } from '@/context/GlobalStateContext';
+import { WalletAccount } from '@cosmos-kit/core';
+import { useWalletClient } from '@cosmos-kit/react';
+import { useEffect, useState } from 'react';
+
+export type CardView =
+  | 'history'
+  | 'historyDetail'
+  | 'destination'
+  | 'alertSubscription'
+  | 'editDestination';
```

Comment:
Do we need the 'editDestination' cardView, it can show as the modal in 'destination' view

Reply:
Well, I think we might not need to have that if it will use popup modal, so I removed it. 
With that being said, feel free add any needed view if needed